bokehjs/src/lib/models/widgets/autocomplete_input.ts

```diff
@@ -139,9 +139,18 @@ export class AutocompleteInputView extends TextInputView {
         }
 
         const completions: string[] = []
+        const case_sensitive: boolean = this.model.case_sensitive
+        let actest: (t: string, v: string) => boolean;
+        if (case_sensitive) {
+          actest = (t: string, v: string): boolean => { return t.startsWith(v); }
+        } else {
+          actest = (t: string, v: string): boolean => { return t.toLowerCase().startsWith(v.toLowerCase()); }
+        }
```

Comment:
Given that `actest` has a type signature, later type repetition isn't necessary. Single line arrow functions can be written more simply without `{}` and `return` statement. We also don't permit usage of semicolons (unless unavoidable by the language grammar). Preferably the code should look like this (barring my other comments):
```ts
        const {case_sensitive} = this.model
        let actest: (t: string, v: string) => boolean
        if (case_sensitive) {
          actest = (t, v) => t.startsWith(v)
        } else {
          actest = (t, v) => t.toLowerCase().startsWith(v.toLowerCase())
        }
```

Reply:
Thanks for your patience, I have written mostly python in the past and almost zero javascript so I appreciate the help!